working_script.R

```diff
@@ -85,37 +84,37 @@ world$map$terrain[world$map$topography == "water"] <- "water"
 
 plot_topography(world) + p
 
+world <- classify_temps(world, breaks = c("polar" = -999,
+                                          "very_cold" = -35,
+                                          "cold" = -15,
+                                          "temperate" = 10,
+                                          "warm" = 25,
+                                          "very_warm" = 33))
```

Comment:
Moving this up here won't work because you haven't run gen_temperature yet, but this is the working_script so it shouldn't matter.

Reply:
So is it that the block below needs to be moved after it instead?

Reply:
You'll find that the block below already exists at the bottom, below the place where you moved this from :sweat_smile: The plot code up here is a copy. You can comment it out or delete. Again, this file being linearly broken isn't very surprising. This is where ideas are flowing.